Windows/MainMenu.xaml.cs

```diff
@@ -24,5 +25,11 @@ public MainMenu()
         {
             InitializeComponent();
         }
+
+        private void openInstellingenUI(object sender, RoutedEventArgs e)
+        {
+            instellingenUI instellingenUI = new instellingenUI();
```

Comment:
dit is vooral het ding dat gefixt is met usercontrol dingen  - deze windows vonden het leuk om zichzelf buiten het programma te openen, maar dit kun je makkelijk aanpassen - zie "opscherm", en hoe ik daar met een toets het scherm verander